themes/digital.gov/src/scss/new/_footer.scss

```diff
@@ -48,6 +48,7 @@ footer.usa-footer {
 
     .social-links {
       @include add-list-reset;
+      margin-top: units(1);
```

Comment:
I noticed a slight `top-margin` spacing difference on the `social-links` column.

**About**
<img width="451" alt="Screen Shot 2024-02-08 at 1 12 20 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/6b187c83-4f32-416d-8b60-126bc3828ecd">

**Social Before**
<img width="478" alt="Screen Shot 2024-02-08 at 1 12 05 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/7ae6f555-5b5b-4d52-b209-4369e966db39">

**Social After**
<img width="477" alt="Screen Shot 2024-02-08 at 1 12 13 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/798ed70b-84fd-4a96-9986-d25323b9e3a9">